libs/nuxt/src/builders/server/schema.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$id": "https://json-schema.org/draft-07/schema",
+  "title": "Server builder",
+  "description": "",
+  "type": "object",
+  "properties": {
+    "browserTarget": {
+      "type": "string",
+      "description": "Target to serve.",
+      "pattern": "^[^:\\s]+:[^:\\s]+(:[^\\s]+)?$"
+    },
+    "watch": {
```

Comment:
How is this option used?

Reply:
It's required by the Cypress builder, but hidden because Nuxt doesn't support not watching in `dev` mode.